packages/csharp/PortingAssistant/PortingAssistant.Telemetry/Models/MetricsType.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace PortingAssistant.Telemetry.Model
+{
+    public enum MetricsType
+    {
+        Project,
+        Solutions,
+        apis,
+        nuget,
```

Comment:
Can these be cosistent? Need to capitalize apis and nuget.. Also, Project/Solution or Projects/Solutions, depending on how it's being used.

Reply:
apis and nuget are lower case to keep consistency with the way currently the metrics are being used in Kibana. Should we capitalize them across everywhere?

Reply:
You can use JsonProperty if they are being serialized to Json to maintain the same names. I dont think we need to capitalize everywhere. Example in this link https://www.newtonsoft.com/json/help/html/JsonPropertyName.htm